book/src/structs.md

```diff
@@ -0,0 +1,24 @@
+# Struct types
+
+Numbat supports nominally typed records:
```

Comment:
I think I would skip the "nominally typed" part, as I'm sure most users will not know what it means:
```suggestion
Numbat supports structs as compound data structures:
```